example_sp/djangosaml2_sp/saml2_sp/templates/amministrazione.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{%block tab_active%} active{%endblock%}
+{% block centered_container %}
+<section id="head-section">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-6 offset-lg-1 order-lg-2">
+          <img src="https://picsum.photos/800/600" title="titolo immagine" alt="descrizione immagine" class="img-fluid">
+        </div>
+        <div class="col-lg-5 order-lg-1">
+          <div class="card">
+            <div class="card-body pb-5">
+              <div class="category-top">
+                <svg class="icon">
+                  <use xlink:href="/bootstrap-italia/dist/svg/sprites.svg#it-calendar"></use>
```

Comment:
```suggestion
                  <use xlink:href="/bootstrap-italia/dist/svg/sprites.svg#it-calendar"></use>
```

please use {% static %}, don't use absolute webpaths

Reply:
here and in all the occurences